phpBB/styles/prosilver/template/ucp_pm_viewfolder.html

```diff
@@ -1,130 +1,117 @@
+				<span class="pm-secondary">
+					<span class="pm-secondary-item pm-authour">
+						{% if message.S_AUTHOR_FOE %}
+							<i class="fa fa-ban fa-fw" title="{{ lang('PM_AUTHOR_FOE', message.MESSAGE_AUTHOR) }}" aria-label="{{ lang('PM_AUTHOR_FOE', message.MESSAGE_AUTHOR) }}"></i>
+						{% else %}
+							<i class="fa fa-user fa-fw" aria-hidden="true"></i>
+						{% endif %}
+
+						{% if S_SHOW_RECIPIENTS %}
+							{{ lang('MESSAGE_TO') }} {{ message.RECIPIENTS }}
+						{% else %}
+							{{ message.MESSAGE_AUTHOR_FULL }}
+						{% endif %}
+					</span>
+					<span class="pm-secondary-item pm-date">
+						<i class="fa fa-clock-o fa-fw" aria-hidden="true"></i>
+						{{ message.SENT_TIME }}
+					</span>
+				</span>
+			</span>
+		</li>
+		{% endfor %}
+	</ul>
+	<div class="pm-viewer">
+		<div class="pm-message">
+			{% if S_TO_RECIPIENT %}
+				<!--Need to figure out how to incorporate this if at all needed-->
+				<strong>{{ lang('TO') }}{{ lang('COLON') }}</strong>
+				{% for recipient in to_recipient %}
+					{% if recipient.NAME_FULL %}
+						{{ recipient.NAME_FULL }}
+					{% else %}
+						<a href="{{ recipient.U_VIEW }}" style="color:{% if recipient.COLOUR %}{recipient.COLOUR}{% elseif recipient.IS_GROUP %}#0000FF{% endif %};">{{ recipient.NAME }}</a>
+					{% endif %}
+				{% endfor %}
+			{% endif %}
+
+			{% for history in history_row %}
+				<ul class="pm-message-list{% if history.USER_ID_NOT_SELF %} pm-not-self{% endif %}">
+					<li class="pm-message-item ">
+						<span class="pm-message-primary">
+							<span class="pm-message-primary-item">
+								<span class="pm-message-authour">
+									<i class="fa fa-user fa-fw" title="{{ message.MESSAGE_AUTHOR }}" aria-label="{{ message.MESSAGE_AUTHOR }}"></i>
+									{{ history.MESSAGE_AUTHOR_FULL }}
+									<!--Maybe add recipient here-->
+									<!-- <i class="fa fa-chevron-right fa-fw" aria-hidden="true"></i> -->
+								</span>
+								<span class="pm-message-date">
+									<i class="fa fa-clock-o fa-fw" title="{{ message.SENT_TIME }}" aria-label="{{ message.SENT_TIME }}"></i>
+									{{ history.SENT_DATE }}
+								</span>
+							</span>
+							<span class="pm-message-secondary">
+								{% if history.S_CURRENT_MSG %}
+									<!--Some way to indicate this is the message clicked, change of color isn't good enough-->
+									<!--<i class="fa fa-certificate fa-fw msg-unread" title="Unread message" aria-label="Unread message"></i>-->
+								{% endif %}
+								<a href="{{ history.U_VIEW_MESSAGE }}" class="pm-message-title{% if history.S_CURRENT_MSG %} pm-message-unread{% endif %}">{{ history.SUBJECT }}</a>
+								<br />
+								{% if history.MESSAGE %}
+									{{ history.MESSAGE }}
+								{% else %}
+									<span class="error">{{ lang('MESSAGE_REMOVED_FROM_OUTBOX') }}</span>
+								{% endif %}
+							</span>
+						</span>
+					</li>
+				</ul>
+				<div class="pm-clear"></div>
```

Comment:
What is this for?

Reply:
![screen shot 2018-07-03 at 8 00 21 pm](https://user-images.githubusercontent.com/25308382/42226110-c3189c78-7efb-11e8-8c7e-4069a3df1b33.png)
After floating to the left, this overlap occurs. I looked it up and found adding the follwing style to the class works

	clear:both
Is there a better way?

Reply:
Yes not to use floats instead we can use flex box, but even with float you apply the style for clearing to a row and wrap it not add arbitrary docs like this. That’s very bad practice